tests/install/openqa_webui.pm

```diff
@@ -81,4 +81,8 @@ sub run {
     clear_root_console;
 }
 
+sub post_fail_hook {
+    # If the installation failed there is no point in collecting logs of the openQA setup
```

Comment:
But the above code includes starting openQA services when it would make very much sense to collect logs